src/main/java/gible/domain/event/entity/Event.java

```diff
@@ -0,0 +1,41 @@
+package gible.domain.event.entity;
+
+import gible.domain.participate.entity.Participate;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @NotNull
+    private String title;
+
+    @NotNull
+    private String content;
+
+    @Column(name = "image_url")
+    private String imageUrl;
+
+    @OneToMany(mappedBy = "event", cascade = CascadeType.REMOVE)
+    private List<Participate> participates;
```

Comment:
해당 부분을 null을 허용하기 보다는 아래와 같이 @Builder.Default를 사용하면 좋을것 같다는 생각이 듭니다!
```suggestion
   @Builder.Default
    private List<Participate> participates = new ArrayList<>();
```